### PR TITLE
add remote sh

### PR DESCRIPTION
**Pull request by gabeduke**
-----

**Checklist for plugin developers:**

- [x] Read the [Plugin Naming Guide](https://github.com/GoogleContainerTools/krew/tree/master/docs/NAMING_GUIDE.md) (for new plugins)
- [x] Verify the installation from URL or a local archive works (`kubectl krew install --manifest=[...] --archive=[...]`)

### CODE REVIEW

**Comment by ahmetb** [comment]
Thanks for your contribution. I gather this plugin is pretty much an interactive chooser for `kubectl exec`, right? Does it have any other features?

If that's the case, I will recommend you to
- use "exec" instead of "remote shell" wording
- use a very clear description field, like the one I just explained above.

**Comment by gabeduke** [comment]
@ahmetb I'm not sure if you meant the tool name itself or just the help language in the krew package info. I've updated the krew package info for now.

**Comment by gabeduke** [comment]
@ahmetb the help language is updated as requested: see [6630bd6](https://github.com/kubernetes-sigs/krew-index/pull/127/commits/6630bd6c1b422e011b2a315f77f3924f90340f38)

**Comment by ahmetb** [comment]
Sorry for the delay here.
What do you think about calling this interactive-exec, exec-interactive or iexec or something creative?

Rsh is a unix tool but it’s not relevant to this plugin, which can be misleading.

**Comment by gabeduke** [comment]
@ahmetb changed to `iexec` as suggested

File: plugins/iexec.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    uri: https://github.com/gabeduke/kubectl-iexec/releases/download/0.1.14/kubectl-iexec_0.1.14_Darwin_x86_64.tar.gz
+    sha256: "e1c5524384fa5db09a84341ebd52bcfaa9d132fa341889c60ccb067335781d77"
+    files:
+    - from: "/kubectl-iexec"
+      to: "."
+    bin: "./kubectl-iexec"
+  shortDescription: Interactive selection tool for `kubectl exec`
+  homepage: https://github.com/gabeduke/kubectl-iexec
+  description: |
+    _          _               _   _        _
```
Comment (ahmetb) [inline plugins/iexec.yaml]: you might want to show your ascii art in `caveats` field, along with some examples perhaps.

right now I'm not very positive about `description` field having monospace art, as we might render them on readme, or an html site perhaps. `caveats` field is shown in the terminal after installing the plugin, so it's more appropriate. see other plugins as a reference.

if I were you, I'd remove ascii art altogether for the sake of simplicity. it doesn't add much to the experience.

File: plugins/iexec.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: iexec
+spec:
+  version: "0.1.9"
```
Comment (ahmetb) [inline plugins/iexec.yaml]: this is not matching the url below.
can you also start this string with `v`?

File: plugins/iexec.yaml (hunk 1 above)
Comment (gabeduke) [reply plugins/iexec.yaml]: Removed as suggested. Fixed in the upstream tool help menu as well.

File: plugins/iexec.yaml (hunk 2 above)
Comment (gabeduke) [reply plugins/iexec.yaml]: Fixed & updated to the latest version

**Pull request merged**